docs/source/fast_training.rst

```diff
@@ -74,6 +74,8 @@ If you don't want to check 100% of the training/validation/test set (for debuggi
         limit_test_batches=0.3
     )
 
+If you also pass ``shuffle=True``, a different random subset of your dataset will be used for each epoch; otherwise the same subset will be used for all epochs.
```

Comment:
Shuffle is not a Trainer argument, but the text clearly refers to that. What you mean is the property in the DataLoader.

Reply:
mind send a fix? :angel: